### ISSUE TITLE
S3 support is broken with 0.7.0.0

### ISSUE THREAD

**Issue by SuperTux88** [diaspora/diaspora#7563]
For more details see: https://discourse.diasporafoundation.org/t/error-when-migrating-to-0-7/868

I have an idea what's going wrong, `CarrierWave::Storage::Fog` is not loaded automatically anymore, so maybe this:
```ruby
config.fog_provider = "fog/aws"
require "carrierwave/storage/fog"
```
in the [initializers/carrierwave.rb](https://github.com/diaspora/diaspora/blob/ac147cc9f438f0d7a374b27a155d8e1fc4952df7/config/initializers/carrierwave.rb#L9) (before `config.storage = :fog`) already helps? But I have no S3 and I can't test it.

The bigger problem with this is: Nobody of the developers uses S3 (as far as I know) and can maintain and test S3 support. So it can break anytime again. Should we simply remove S3 support or does anybody want to take care of it?

**Comment by trek-e** [comment]
adding these two lines to the file corrected the error I was experiencing.

**Comment by SuperTux88** [comment]
> adding these two lines to the file corrected the error I was experiencing.

Thanks for testing the fix, so I created #7566.

> I'd be happy to test as I have s3 turned on my little pod but I'm not a developer so probably couldn't maintain it.

Testing every release before it's released (so during the freeze time, which is usually one week before the release) and then report when something breaks would at least be a start. So we can make sure the released version works.

Maybe @jramos can help here too, because he already worked on the S3 support in the past and I think he uses it too. But we need at least somebody who uses it before the release, so we can make sure the released version will work.

**Comment by AutumnFox** [comment]
The issue is affect the upgrading process as well - there is an fog non initialized exception while doing DB migration RAILS_ENV=production bin/rake db:migrate.

Fortunately the proposed solution successfully fix it.
